chore: 264, 265, 266, 270

HoomanDigital opened the pull request:
#264 #265 #266 #270

aktdenis commented:
thanks! 

@ArthurEmma2 can we get rid off the white outline in the second and third image. also, there is no need for the line(border) below CTA Launch... 

cc @HoomanHQ 

<img width="1352" alt="Screenshot 2023-10-11 at 13 04 57" src="https://github.com/akash-network/website/assets/116030971/00a059d6-3779-4cb4-b06b-cd1393790b1a">

aktdenis commented:
@ArthurEmma2  could you please have a look at the icon here. seems outlined and not aligned properly. thanks!
<img width="315" alt="Screenshot 2023-10-11 at 13 57 59" src="https://github.com/akash-network/website/assets/116030971/ed3e4cac-e3d0-42e8-a025-201c4d0acfe3">

ArthurEmma2 commented:
> thanks!
> 
> @ArthurEmma2 can we get rid off the white outline in the second and third image. also, there is no need for the line(border) below CTA Launch...
> 
> cc @HoomanHQ
> 
> <img alt="Screenshot 2023-10-11 at 13 04 57" width="1352" src="https://user-images.githubusercontent.com/116030971/274243807-00a059d6-3779-4cb4-b06b-cd1393790b1a.png">

the white outline is part of the svg it's not from my side

ArthurEmma2 commented:
@aktdenis new update pushed and deployed check it out

dharamveergit committed:
chore: Update top-pagination SVG dimensions for better display

dharamveergit committed:
feat: Update GPU rental link with model details

The GPU rental link in the GPU table has been updated to include the vendor, model, interface, and VRAM details in the URL query parameters. This change ensures that the user is directed to the correct GPU rental page with the specific GPU model pre-selected.

dharamveergit committed:
Update image file extensions for deployment guides

Commit:
Merge branch 'main' into dharamveer

Commit:
Merge pull request #1 from HoomanDigital/dharamveer

Fix #264 , #265, #266
